docs/metadata-ui.rst

```diff
@@ -0,0 +1,190 @@
+
+- Activate the virtual environment.
+
+- Run::
+    
+    $ pip install omero-metadata
+
+**Populate Metadata script**
+
+No explicit installation necessary, shipped with the OMERO.server.
+
+
+Step-by-step
+------------
+
+.. _Populatemetadatapdi:
+
+*Project/Dataset/Image*
+~~~~~~~~~~~~~~~~~~~~~~~
+
+#.  Log in to OMERO.web, create a new Dataset and copy into it four images, preferably images which have no OMERO.tables on them. Note the name of the images you are copying in.
+
+    |image4|
+
+#.  Download :download:`four-images.csv <../scripts/four-images.csv>`. Open the CSV file in Excel and edit the name of the images in the first column to match the names of the images you copied into your Dataset in the previous step. Also, edit the name of the Dataset in the second column to match the name of your Dataset in OMERO.web. Save the file locally as CSV.
+
+#.  (Optional) In your OMERO.web, upload the CSV file you just saved and attach it onto the Dataset you created previously. Alternatively, you can skip this step, and point the ``Populate Metadata`` script to the local CSV, as explained below.
+
+    |image5|
+
+#.  Select the Dataset you created and attached to it the CSV. Find the script icon |image6| above the central pane, expand it and find the ``Import scripts`` section. In there, select the ``Populate metadata`` script.
+
+    |image7|
+
+#.  If you did not attach the CSV to the Dataset, you can now click on the ``Browse`` button and select the CSV from your local machine.
+
+#.  Run the script.
+
+#.  Click again onto the Dataset in the left-hand pane to refresh and observe that there is a new Attachment in the right hand pane under "Attachments" harmonica, named ``bulk_annotations``. 
+
+    |image8|
+
+#.  Click on single images inside the Dataset and observe that in the "Tables" harmonica in the right-hand pane there are new values coming originally from your edited CSV.
+
+    |image9|
+
+.. _Populatemetadataspw:
+
+*Screen/Plate/Well*
+~~~~~~~~~~~~~~~~~~~
+
+#.  Find a Plate inside a Screen in OMERO.web which has no OMERO.tables on its Wells.
+
+#.  Download :download:`simple-screen.csv <../scripts/simple-screen.csv>`. Open the CSV file in Excel and edit the name of the wells in the first column to match the names of the wells in your Plate from the previous step. Also, edit the name of the Plate in the second column to match the name of your Plate in OMERO.web. Save the file locally as CSV.
+
+#.  (Optional) In your OMERO.web, upload the CSV file you just saved and attach it onto the Screen containing the Plate you created previously. Alternatively, you can skip this step, and point the ``Populate Metadata`` script to the local CSV, as explained below.
+
+    |image5b|
+
+#.  Select the Screen you identified above and (optionally) attached to it the CSV. Find the script icon |image6| above the central pane, expand it and find the ``Import scripts`` section. In there, select the ``Populate metadata`` script.
```

Comment:
Again, although you have 'optionally' here, you can still simplify by removing "and (optionally) attached to it the CSV". You only have the 1 Screen.

Reply:
fixed in https://github.com/ome/omero-guide-upload/pull/13/commits/282d03dd8767cbf3b92a94b54aa4ddc6b74bb425